core/domain/services/admin/privacy/policy/PrivacyPolicy.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace EventEspresso\core\domain\services\admin\privacy\policy;
+
+use EEH_Template;
+use EventEspresso\core\services\privacy\policy\PrivacyPolicyInterface;
+
+defined('EVENT_ESPRESSO_VERSION') || exit('No direct script access allowed');
```

Comment:
you'll need to run CodeSniffer on this file because the above line (as well as some of the spaces below it) are no longer kosher and need to be removed.

Reply:
kk!